apps/src/lib/ui/PopUpMenu.story.jsx

```diff
@@ -40,9 +40,9 @@ class Overview extends Component {
           It targets the bottom-center of this element.
         </div>
         <PopUpMenu isOpen targetPoint={this.state.targetPoint}>
-          <PopUpMenu.Item>Option One</PopUpMenu.Item>
-          <PopUpMenu.Item>Option Two</PopUpMenu.Item>
-          <PopUpMenu.Item>Option Three</PopUpMenu.Item>
+          <PopUpMenu.Item onClick={() => {}}>Option One</PopUpMenu.Item>
```

Comment:
I don't see these as required in props, why do we need to add these here?

Reply:
They aren't marked as required, but the item will through an exception if an href/onclick is not provided.